packages/react-button/src/components/ToggleButton/stories/ToggleButtonChecked.stories.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
-import { ToggleButton } from '../../../ToggleButton'; // codesandbox-dependency: @fluentui/react-button ^9.0.0-beta
+import { ToggleButton } from '../../../ToggleButton';
 
 export const Checked = () => (
   <>
-    <ToggleButton checked={true}>A checked toggle button</ToggleButton>
+    <ToggleButton checked={true}>Checked</ToggleButton>
```

Comment:
Similar to the "disabled" button label, I think having the label just be "checked" or "unchecked" might be a little weird b/c the state will also be announced by screen readers. Maybe something like this?:
```suggestion
    <ToggleButton checked={true}>Controlled checked state</ToggleButton>
```